Marlin/src/lcd/tft/ui_480x320.cpp

```diff
@@ -623,6 +623,7 @@ struct {
 #define BTN_HEIGHT 52
 #define X_MARGIN 20
 #define Y_MARGIN 15
+#define Y_OFFSET TERN(HAS_UI_480x272, 28, 34)
```

Comment:
e.g., Maybe we should just be using `FONT_LINE_HEIGHT` which may be 34, 26, or 24 depending on the UI height and the selected `TFT_FONT`.